src/main/java/io/supertokens/storageLayer/StorageLayer.java

```diff
@@ -245,6 +246,15 @@ public static void loadAllTenantStorage(Main main, TenantConfig[] tenants)
                     if (idToExistingStorageLayerMap.containsKey(uniqueId)) {
                         // we reuse the existing storage layer
                         resourceKeyToStorageMap.put(key, idToExistingStorageLayerMap.get(uniqueId).storage);
+                    } else {
+                        try {
+                            // Creating new storage here with logging enabled
+                            resourceKeyToStorageMap.put(
+                                    key, StorageLayer.getNewStorageInstance(
+                                            main, normalisedConfigs.get(key), key.getTenantIdentifier(), false));
+                        } catch (InvalidConfigException e) {
+                            throw new RuntimeException(e); // Should never come here
+                        }
```

Comment:
```suggestion
                    
```

Reply:
- Make sure that if we do the above, some test fails
- Then remove the above, and replace with just setting the right log level